package.json

```diff
@@ -22,7 +22,7 @@
     "react-dom": "16.13.1",
     "react-query": "^2.5.6",
     "react-router-dom": "5.1.2",
-    "react-scripts": "3.4.1",
+    "react-scripts": "^4.0.0",
```

Comment:
Is there a reason for this version upgrade?
Version 4.0.0 is breaking tests, I think you should return to version 3.4.1 by now. Then you will be able to run `npm test` and fix any failing test.

Reply:
@isakawa-carbon thanks for the feedback on this. @Julhinha I do want to upgrade to 4.0 soon here (mainly for fast refresh support again) but that will likely be scheduled for the first thing after this release